src/sentry/grouping/strategies/configurations.py

```diff
@@ -124,32 +97,23 @@ def register_strategy_config(id: str, **kwargs) -> type[StrategyConfiguration]:
     enhancements_base="common:2019-03-23",
 )
 
+# This is the grouping strategy used for new projects.
 register_strategy_config(
-    id="newstyle:2019-10-29",
+    id="newstyle:2023-01-11",
     base="newstyle:2019-05-08",
-    risk=RISK_LEVEL_MEDIUM,
+    risk=RISK_LEVEL_HIGH,
     changelog="""
         * Better rules for when to take context lines into account for
           JavaScript platforms for grouping purposes.
         * Better support for PHP7 anonymous classes.
-    """,
-    initial_context={
-        "php_detect_anonymous_classes": True,
-        "with_context_line_file_origin_bug": False,
```

Comment:
These will be moved to the latest configuration.
<img width="484" alt="image" src="https://github.com/user-attachments/assets/bbe0c6fe-799b-4ea7-b359-8f108eab93b1">